src/icons/svgs/essentials/icon-break.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M8 9.01952V3" stroke="#F59140" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M8 12.6553V13.0001" stroke="#F59140" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
We need to replace these `stroke` values with `#000` so that we can change the color programatically.

I really need to write a guide on how to add new icons. If you check out the deploy preview icons you can see that these are always yellow even when you try to set them to teal.
```suggestion
<path d="M8 9.01952V3" stroke="#000" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
<path d="M8 12.6553V13.0001" stroke="#000" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
```